src/components/approval-type/ApprovalType.vue

```diff
@@ -53,78 +92,100 @@ export default class ApprovalType extends Vue {
   }
 
   /** Draft court order number. */
-  @Prop({ default: '' }) readonly draftCourtOrderNumber!: string
+  @Prop({ default: '' }) readonly courtOrderNumber!: string
 
   /** Whether approved by the registrar. */
-  @Prop({ default: false }) readonly draftApprovedByRegistrar!: boolean
+  @Prop({ default: false }) readonly approvedByRegistrar!: boolean
 
   /** filing name used in radio options. */
   @Prop({ default: 'restoration' }) readonly filingType!: string
 
   /** Show only the court order option; remove via registrar option. */
   @Prop({ default: false }) readonly isCourtOrderOnly!: boolean
 
+  /** Draft notice date. */
+  @Prop({ default: '' }) readonly noticeDate!: string
+
+  /** Draft application date. */
+  @Prop({ default: '' }) readonly applicationDate!: string
+
   // Local properties
-  private courtOrderNumber = ''
-  private courtOrderNumRules = []
+  private courtOrderNumberText = ''
   private valid = false
   private approvalTypeSelected = ''
+  private noticeDateText = ''
+  private applicationDateText = ''
+
+  // Date Picker Rules
+  private datePickerRules = [(v: string) => !!v || 'Date is required']
+
+  // Text Field Rules
+  private courtOrderNumRules = [
```

Comment:
Not serious enough to change, but these don't have to be private -- external code could read these without causing an issue or seeing anything secret -- but they should be readonly so code doesn't try to reassign these.